ods_ci/tasks/Resources/Provisioning/Hive/AWS/aws-cluster.yaml

```diff
@@ -52,6 +52,7 @@ items:
         networkType: {{ ocp_network_type|default("OVNKubernetes") }}
         serviceNetwork:
         - 172.30.0.0/16
+      fips: true
```

Comment:
As meentioned in the call. Would you mind not setting the FIPS value to true? It would be more beneficial to set the FIPS value to false and adjust it dynamically based on the user's input if they desire a FIPS cluster. Controlling it dynamically would be a preferable approach in my opinion.

Could you try something like below:
` fips: {{fips_validation|default("false") }}`